setup.cfg

```diff
@@ -46,9 +47,13 @@ install_requires =
     invenio-stats>=4.0.0,<5.0.0
     invenio-vocabularies>=2.0.0,<3.0.0
     nameparser>=1.1.1
+    pycountry>=22.3.5
+    pydash>=6.0.0,<7.0.0
     pytz>=2020.4
     pyyaml>=5.4.0
     python-slugify>=8.0.1
+    # Upper pinning due to commonmeta-py
+    types-requests<2.31.0.7
```

Comment:
For reviewers:

It happens that `commonmeta-py` has a dependency on `types-requests` that is not upper pinned. Therefore, it ends up installing its latest which depends on `urllib3 >= 2` and that clashes with rdm dependencies (namely search dependencies). 

I pinned it here since it would be safer for RDM this way, however this might happen with other dependencies of `commonmeta-py` in the future. Please let me know if this is not the right way to do it

Reply:
It's right, and I think we don't depend on any of the `requests`/network-related features of `commonmeta-py`.